Wrappers/Python/cil/optimisation/operators/GradientOperator.py

```diff
@@ -32,64 +32,72 @@
 
 class GradientOperator(LinearOperator):
 
-
-    r'''Gradient Operator: Computes first-order forward/backward differences on 
-        2D, 3D, 4D ImageData under Neumann/Periodic boundary conditions
-    
-    :param gm_domain: Set up the domain of the function
-    :type gm_domain: `ImageGeometry`
-    :param bnd_cond: Set the boundary conditions to use 'Neumann' or 'Periodic', defaults to 'Neumann'
-    :type bnd_cond: str, optional    
-    :return: returns a BlockDataContainer containing images of the derivatives order given by `dimension_labels`\
+    r"""
+    Gradient Operator: Computes first-order forward/backward differences on
+    2D, 3D, 4D ImageData under Neumann/Periodic boundary conditions
+
+    Parameters
+    ----------
+    domain_geometry: ImageGeometry
+        Set up the domain of the function
+    method: str, default 'forward'
+        Accepts: 'forward', 'backward', 'centered', note C++ optimised routine only works with 'forward'
+    bnd_cond: str, default,  'Neumann'
+        Set the boundary conditions to use 'Neumann' or 'Periodic'
+    **kwargs:
+        correlation: str, default 'Space'
+            'Space' will compute the gradient on only the spatial dimensions, 'SpaceChannels' will include the channel dimension direction
+        backend: str, default 'c'
+            'c' or 'numpy', defaults to 'c' if correlation is 'SpaceChannels' or channels = 1
+        num_threads: int
+            If backend is 'c' specify the number of threads to use. Default is number of cpus/2          
+        split: boolean
+            If 'True', and backend 'C' will return a BlockDataContainer with grouped spatial domains. i.e. [Channel, [Z, Y, X]], otherwise [Channel, Z, Y, X]
```

Comment:
```suggestion
            If 'True', and backend 'c' will return a BlockDataContainer with grouped spatial domains. i.e. [Channel, [Z, Y, X]], otherwise [Channel, Z, Y, X]
```
looks to me like backend only allows lowercase not uppercase c